__tests__/authRoutes.test.js

```diff
@@ -67,9 +64,6 @@ describe('test for user login', () => {
         expect(res.status).to.eql(200);
         expect(res.body.message).to.eql('You have successfully logged in');
         expect(res.body.user).to.have.property('token');
-        expect(res.body.user).to.have.property('isVerified');
-        expect(res.body.user).to.have.property('email');
-        expect(res.body.user.email).to.eql(mockUsers[6].email);
         done();
```

Comment:
I have a suggestion. Instead of taking these specific assertions out, can we decode the token to confirm that we have the correct info?

Eg:
`expect(decodedToken.email).to.eql(mockUsers[6].email)`? etc etc

Of course we'll make this change everywhere we removed similar assertions.

Reply:
Yes, that makes a lot of sense, I'll do that. Thanks